students/kristoffer_jonson/lesson4/mailroom.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import sys
+from datetime import date
+today = date.today()
+
+donor_db = {"William": [653772.32, 12.17],
+            "Jeff": [877.33],
+            "Paul": [663.23, 43.87, 1.32],
+            "Mark": [1663.23, 4300.87, 10432.0],
+            "Elon": [234.25, 2764.87, 9783.0],
+            }
+response = ''
+
+def main(donor_db):
+    """
+    Prompts user for navigation of donor database
+    :param donor_db: Database of donations and donors
+    """
+    response = ''
+
+    switch_func_dict = {
+        '1':thank_you,
+        '2':create_report,
+        '3':all_thank_you,
+        '4':exit_program
+    }
+
+    while True:
+        response = initial_prompt()
+        switch_func_dict.get(response,"nothing")(donor_db)
```

Comment:
your donor_db structure is global so you don't need to pass it into each function